docs/concepts/automations.md

```diff
@@ -310,6 +314,86 @@ However, without `for_each`, a `prefect.flow-run.Completed` event from a _differ
+* **Lateness** (`{"name": "lateness"}`), defined as the _length of time_ that a `Scheduled` flow remains in a `Late` state before transitioning to a `Running` and/or `Crashed` state.  Only flow runs that the system marks `Late` are included.
+
+The schema of a metric trigger is as follows:
+
+| Name               | Type               | Supports trailing wildcards | Description |
+| ------------------ | ------------------ | --------------------------- | ----------- |
+| **match**          | object             | :material-check:            | Labels for resources which this Automation will match. |
+| **match_related**  | object             | :material-check:            | Labels for related resources which this Automation will match. |
+| **metric**         | `MetricTriggerQuery`        | N/A                         | The definition of the metric query to run |
+
+And the `MetricTriggerQuery` query is defined as:
+
+| Name               | Type                 |  Description |
+| ------------------ | -------------------- |  ----------- |
+| **name**          | string               |  The name of the Prefect metric to evaluate (see above). |
+| **threshold**  | number               |  The threshold to which the current metric value is compared |
+| **operator**         | string (`"<"`, `"<="`, `">"`, `">="`) | The comparison operator to use to decide if the threshold value is met |
+| **range**         | duration in seconds | How far back to evaluate the metric |
+| **firing_for**         | duration in seconds | How long the value must exceed the threshold before this trigger fires |
+
+For example, to fire when flow runs tagged `production` in your workspace are failing at a rate of 10% or worse for the last hour (in other words, your success rate is below 90%), create this trigger:
+
+```json
+{
+  "type": "metric",
+  "match": {
+    "prefect.resource.id": "prefect.flow-run.*"
+  },
+  "match_related": {
+    "prefect.resource.id": "prefect.tag.production",
+    "prefect.resource.role": "tag"
+  },
+  "metric": {
+    "name": "successes",
+    "threshold": 0.9,
+    "operator": "<",
+    "range": 3600,
+    "firing_for": 0
+  }
+}
+```
+
+To detect when the average lateness of your Kubernetes workloads (running on a work pool named `kubernetes`) in the last day exceeds 5 minutes late, and that number hasn't gotten better for the last 10 minutes, use a trigger like this:
+
+```json
+{
+  "type": "metric",
+  "match": {
+    "prefect.resource.id": "prefect.flow-run.*"
+  },
+  "match_related": {
+    "prefect.resource.id": "prefect.work-pool.kubernetes",
+    "prefect.resource.role": "work-pool"
+  },
+  "metric": {
+    "name": "lateness",
+    "threshold": 300,
+    "operator": ">",
+    "range": 86400,
+    "firing_for": 600
+  }
+}
+```
+
+### Composite triggers
+
+[more information coming soon]
```

Comment:
@serinamarie here's the placeholder I dropped for composite triggers

Reply:
❤️ Cool, would it make sense for folks to make a branch into this one, or update this branch directly?

Reply:
Either way, I still have a few things to tidy up tomorrow so feel free to push here in the meantime